pkg/cli/cluster_free_disk_space.go

```diff
@@ -0,0 +1,125 @@
+package cli
+
+import (
+	"bytes"
+	"fmt"
+	"log"
+
+	"github.com/spf13/cobra"
+	v1 "k8s.io/api/storage/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+
+	rookcli "github.com/rook/rook/pkg/client/clientset/versioned"
+
+	clusterspace "github.com/replicatedhq/kurl/pkg/cluster/space"
+)
+
+const (
+	IsDefaultStorageClassAnnotation = "storageclass.kubernetes.io/is-default-class"
+	OpenEBSLocalProvisioner         = "openebs.io/local"
+	RookRBDProvisioner              = "rook-ceph.rbd.csi.ceph.com"
+	RookCephFSProvisioner           = "rook-ceph.cephfs.csi.ceph.com"
+)
+
+func NewClusterFreeDiskSpaceCmd(cli CLI) *cobra.Command {
+	var forStorageClass, openEBSImage, openEBSNode string
+	logbuf := bytes.NewBuffer(nil)
+	logger := log.New(logbuf, "", 0)
+	k8sConfig := config.GetConfigOrDie()
+	clientSet := kubernetes.NewForConfigOrDie(k8sConfig)
+	rookClientSet := rookcli.NewForConfigOrDie(k8sConfig)
+
+	cmd := &cobra.Command{
+		Use:          "free-disk-space",
+		Short:        "Returns the amount of free disk space (in bytes) for a given storage class",
+		SilenceUsage: true,
+		RunE: func(cmd *cobra.Command, args []string) (err error) {
```

Comment:
Could we add an Example desc with example over how to call it? Also, could we add an Long desc describing when/why/it is required useful?  

See an example
https://github.com/kubernetes-sigs/kubebuilder/blob/master/pkg/plugins/golang/v3/init.go#L64-L76

Reply:
I have added and example to the `Example` property. Is that enough ?